test/form/samples/render-declaration-semicolons/_expected/system.js

```diff
@@ -22,13 +15,13 @@ System.register('bundle', [], function (exports) {
 			const g = 3, h = 4;
 			console.log(g, h);
 
-			var i, j;
+			var i, j; exports({ i: i, j: j });
 
-			var k, l;
+			var k, l; exports({ k: k, l: l });
```

Comment:
As much as possible it is usually a good idea to bulk together calls to the `exports({})` function. This is because when there are `export *` or reexports in play, there is a cascade of function calls to propagate the binding updates, so fewer cascades are better for performance as this can degrade on very exported graphs otherwise.

Reply:
Undefined bindings are an exception, but this reduces quite a bit complexity. On the other hand, there is now much more grouping for declarations that contain more than one export, see below lines 22 and 24.